crates/rspack_plugin_javascript/src/plugin/inner_graph_plugin.rs

```diff
@@ -97,6 +98,27 @@ impl<'a> Visit for InnerGraphPlugin<'a> {
     program.visit_children_with(self);
   }
 
+  // TODO: adapt `InnerGraphPlugin` to `ParserPlugin`
+  fn visit_if_stmt(&mut self, n: &swc_core::ecma::ast::IfStmt) {
+    n.test.visit_children_with(self);
+    if !self.path_ignored_spans.is_span_ignored(n.cons.span()) {
+      n.cons.visit_children_with(self);
+    }
+    if let Some(alt) = &n.alt
+      && !self.path_ignored_spans.is_span_ignored(alt.span())
+    {
+      alt.visit_children_with(self)
+    }
+  }
+
+  // TODO: adapt `InnerGraphPlugin` to `ParserPlugin`
+  fn visit_bin_expr(&mut self, n: &swc_core::ecma::ast::BinExpr) {
+    n.left.visit_children_with(self);
+    if !self.path_ignored_spans.is_span_ignored(n.right.span()) {
+      n.right.visit_children_with(self)
+    }
+  }
```

Comment:
This part is for following the ignored path generated by `ConstPlugin` in conditional expressions like `if` or `binary_op`.